src/main/java/duke/Deadline.java

```diff
@@ -0,0 +1,117 @@
+package duke;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
+public class Deadline extends Task {
+
+    private final LocalDate date;
+    private final String time;
+
+    /**
+     * A constructor to initialize a duke.Deadline object.
+     *
+     * @param isDone A boolean to indicate if the deadline is completed.
+     * @param deadlineDescription A string to describe the deadline.
+     * @param index An integer to indicate the position of the duke.Deadline in list of tasks.
+     * @param date A String to indicate the date to be completed by.
+     */
+    public Deadline(boolean isDone, String deadlineDescription, int index, LocalDate date, String time) {
+        super(isDone, deadlineDescription, index);
+        this.date = date;
+        this.time = time;
+    }
+
+    /**
+     * A method to update the deadline from incomplete to complete.
+     */
+    public void markDone() {
+        super.markDone();
+    }
+
+    /**
+     * A method to update the deadline from complete to incomplete.
+     */
+    public void markUndone() {
+        super.markUndone();
+    }
+
+    /**
+     * A method to update the index of the deadline in the list of tasks.
+     *
+     * @param newIndex The most current integer index of the task.
+     */
+    @Override
+    public void setIndex(int newIndex) {
+        super.setIndex(newIndex);
+    }
+
+    /**
+     * Output the deadline added to the list of tasks in the console.
+     */
+    @Override
+    public void printAdded() {
+        System.out.println(Ui.straightLine + "\n  Yep, it's in!\n    [D][ ] " + this.getDescription() + " (by: " +
+                            date.format(DateTimeFormatter.ofPattern("d MMM yyyy")) + " " + time + ")\n  " +
+                            this.getIndex() + " tasks left, 頑張れ!\n" + Ui.straightLine + "\n");
```

Comment:
Perhaps the line break should be before the operator (e.g. the `+` operator), according to [Java coding standards](https://se-education.org/guides/conventions/java/intermediate.html)
```suggestion
        System.out.println(Ui.straightLine + "\n  Yep, it's in!\n    [D][ ] " + this.getDescription() + " (by: "
                            + date.format(DateTimeFormatter.ofPattern("d MMM yyyy")) + " " + time + ")\n  "
                            + this.getIndex() + " tasks left, 頑張れ!\n" + Ui.straightLine + "\n");
```
I noticed the same issue in several other places too.